jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -185,16 +181,38 @@ protected void doStart() throws Exception {
       setConfigurationClasses(quickstartConfigurationClasses);
     }
 
-    datastoreService = getDatastoreService();
-
     if (quickstartWebXml == null) {
       addEventListener(new ContextListener());
     } else {
       getMetaData()
           .addDescriptorProcessor(preconfigProcessor = new PreconfigureDescriptorProcessor());
     }
 
+    // DatastoreServiceFactory.getDatastoreService();
```

Comment:
You can probably remove the related import, however can you explain why is that necessary?